modules/loader.js

```diff
@@ -622,49 +622,70 @@ Loader.prototype.getNetwork = function (cb) {
 	if (__private.network.height > 0 && Math.abs(__private.network.height - modules.blocks.getLastBlock().height) === 1) {
 		return setImmediate(cb, null, __private.network);
 	}
-	async.waterfall([
-		function (waterCb) {
-			modules.transport.getFromRandomPeer({
-				api: '/list',
-				method: 'GET'
-			}, function (err, res) {
-				if (err) {
-					return setImmediate(waterCb, err);
-				} else {
-					return setImmediate(waterCb, null, res);
-				}
-			});
-		},
-		function (res, waterCb) {
-			library.schema.validate(res.body, schema.getNetwork.peers, function (err) {
-				var peers = modules.peers.acceptable(res.body.peers);
+	return library.config.syncPeers.list.length ? setNetwork(library.config.syncPeers.list) : findDecentralizedPeers();
```

Comment:
Maybe we should throw an error when somebody specified empty/incorrect peers to sync with via command line?